Doxyfile

```diff
@@ -811,7 +811,7 @@ WARN_FORMAT            = "$file:$line: $text"
 # messages should be written. If left blank the output is written to standard
```

Comment:
orthogonal to your change but I'm curious if this file can be moved under the eng directory to help cut down on the pollution in the root repo.

Reply:
The actual warning file (i.e. `DoxygenWarningLog.txt`), or this `Doxyfile` file, or both?

I can certainly move the warning file to  `eng\DoxygenWarningLog.txt`

Reply:
I was referencing the DoxyFile but the warning file should probably go wherever we put normal logs for other build logs.

Reply:
> orthogonal to your change but I'm curious if this file can be moved under the eng directory to help cut down on the pollution in the root repo.

https://github.com/Azure/azure-sdk-for-c/issues/1146